intel_extension_for_transformers/llm/runtime/graph/tensor_parallelism.md

```diff
@@ -25,6 +26,28 @@ The scenario for the attention module is more complex. As shown in the following
 ## Prerequisites
 Multi-node and Multi-socket communications are needed in tensor parallelism, we use oneCCL for the distributed communications. 
 
+## Enable Customized Model
+Taking "llama" as an example, we need three modifications:
+
+- First, we need to split the weight used in the 'matmul' calculation.
+- Second, determine the size of 'n_head' after splitting, based on the number of nodes in parallel computation(world_size).
+- Finally, insert the 'all_reduce' operator where necessary.
+
+For instance, when splitting the FFN (feed-forward network) module, it is visible in the FFN flowchart that both 'feedforward.w1.weight' and 'feedforward.w3.weight' are part of the first 'matmul' computation. This portion of the 'matmul' should be split by column, considering the weight in ITREX is already transposed. Here, we only need to set the 'split_type' of these two weights to 'TP_1D_ROW' within the 'calc_split_type()' function in 'models/modelutils/model_files.h'. 
+The 'feedforward.w1.weight' belongs to the second part of the 'matmul' calculation in the FFN process diagram, the 'split_type' should be set to 'TP_1D_COLUMN'. This ensures that the partial results from the first 'matmul' calculation can be independently used for the second 'matmul' calculation. There are also some primitives between the two 'matmul' calculations, and since these primitives are element-wise, they are also calculated independently on their respective nodes.
```

Comment:
variables better follow coding style
```suggestion
The ```feedforward.w1.weight``` belongs to the second part of the ```matmul``` calculation in the FFN process diagram, the ```split_type``` should be set to ```TP_1D_COLUMN```. This ensures that the partial results from the first ```matmul``` calculation can be independently used for the second ```matmul``` calculation. There are also some primitives between the two ```matmul``` calculations, and since these primitives are element-wise, they are also calculated independently on their respective nodes.
```